website/source/docs/configuration/from-1.5/functions/file/templatefile.html.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
This is the doc for the function that is commented out in functions.go, right?

Reply:
Yes, removing any references.